services/src/main/java/org/exoplatform/webconferencing/externalvisio/rest/ExternalVisioConnectorRest.java

```diff
@@ -141,4 +155,37 @@ public Response updateExternalVisioConnector(@Parameter(description = "External
       return Response.serverError().entity(e.getMessage()).build();
     }
   }
+
+  @GET
+  @RolesAllowed("users")
+  @Path("{identityId}")
+  @Produces(MediaType.APPLICATION_JSON)
+  @Operation(summary = "Retrieves the list of configured external visio connectors", method = "GET", description = "Retrieves the list of configured external visio connectors")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+      @ApiResponse(responseCode = "400", description = "Invalid query input"),
+      @ApiResponse(responseCode = "401", description = "Unauthorized"),
+      @ApiResponse(responseCode = "500", description = "Internal server error"), })
+  public Response getConfiguredExternalVisioConnectors(@Parameter(description = "identityId", required = true)
+  @PathParam("identityId")
+  String identityId) {
+    String authenticatedUser = ConversationState.getCurrent().getIdentity().getUserId();
+    Space space = spaceService.getSpaceByPrettyName(identityId);
+    Identity identity = null;
+    if (space != null) {
+      identity = identityManager.getOrCreateSpaceIdentity(identityId);
+      if (!spaceService.isMember(space, authenticatedUser) && !spaceService.isSuperManager(authenticatedUser)) {
+        return Response.status(Response.Status.UNAUTHORIZED).build();
+      }
+    } else {
+      identity = identityManager.getOrCreateUserIdentity(identityId);
+    }
+    try {
+      List<ExternalVisioConnector> externalVisioConnectors =
+                                                           externalVisioConnectorService.getConfiguredExternalVisioConnectors(identity, space.getId());
```

Comment:
Why do you keep space.getId ? 
```suggestion
                                                           externalVisioConnectorService.getConfiguredExternalVisioConnectors(identity);
```